core/src/test/scala/unit/kafka/server/epoch/LeaderEpochIntegrationTest.scala

```diff
@@ -277,6 +283,22 @@ class LeaderEpochIntegrationTest extends QuorumTestHarness with Logging {
     producer.close()
   }
 
+  private def createTopicWithAssignment(topic: String, partitionReplicaAssignment: collection.Map[Int, Seq[Int]]): Unit = {
```

Comment:
Is it possible to place this method inside the `TestUtils`?

Reply:
@ex172000 Of course, I did.

Reply:
@ex172000 
In `TestUtils`, How about changing the method names from `createTopicWithAssignment` to `createTopic`, and from `createTopic` to `createTopicWithZkClient`, similar to `createTopicWithAdmin`?